cache/src/main/java/org/buffer/android/boilerplate/cache/model/ArticleEntity.kt

```diff
@@ -2,14 +2,13 @@ package org.buffer.android.boilerplate.cache.model
 
 import android.arch.persistence.room.Entity
 import android.arch.persistence.room.PrimaryKey
-import org.buffer.android.boilerplate.cache.db.constants.ArticleConstants
-import org.buffer.android.boilerplate.cache.db.constants.BufferooConstants
+import org.buffer.android.boilerplate.cache.dao.TABLE_NAME
 
 /**
  * Model used solely for the caching of a article
  */
-@Entity(tableName = ArticleConstants.TABLE_NAME)
-data class CachedArticle(
+@Entity(tableName = TABLE_NAME)
+data class ArticleEntity(
```

Comment:
It is safe to use `CachedArticle` I guess, but for in the future, this module will be merged to `data` module